.github/workflows/scan-build.yml

```diff
@@ -0,0 +1,29 @@
+name: scan-build
+
+on: [push, pull_request]
+
+jobs:
+  scan-build:
+
+    strategy:
+      matrix:
+        bits: [32, 64]
+        lto: [LTO=y, LTO=n]
+
+    runs-on: 'ubuntu-20.04'
+
+    steps:
+    - name: Install
+      run: |
+        sudo apt-get update -q
+        sudo apt-get install clang-tools-11
+
+    - uses: actions/checkout@v2
+
+    - name: Submodules
+      run: |
+        git submodule update --init tpmlib
+
+    - name: make
+      run: |
+        scan-build-11 --status-bugs -analyze-headers make CC=${{matrix.compiler}} 32=${{matrix.bits == 32 && 'y' || 'n'}} ${{matrix.lto}}
```

Comment:
Just a quick question. The CC  will be empty here, since I do not see any compiler variables in the matrix. Is it intentional?

Reply:
Oh - good point.  That was stale copy&paste.  scan-build swaps clang in in the background